tests/templates/kuttl/smoke/30-install-superset.yaml.j2

```diff
@@ -36,6 +36,14 @@ spec:
     config:
       logging:
         enableVectorAgent: {{ lookup('env', 'VECTOR_AGGREGATOR') | length > 0 }}
+    configOverrides:
+      superset_config.py:
```

Comment:
Hm, looks like there was a misunderstanding. I meant testing explicitely that `EXPERIMENTAL_FILE_HEADER` and his friend can be overridden with `configOverrides` even if they are special cased by the operator.

The general case is already tested in #531 and it's no need to be repeated here.